common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -880,6 +886,44 @@ def get_notification_text(self):
         return self.q(css="{} h3".format(notification_selector)).text
 
 
+class MembershipPageBetaTesterSection(PageObject):
+    """
+    Beta tester section of the Membership tab of the Instructor dashboard.
+    """
+    url = None
+
+    batch_beta_tester_heading_selector = '#heading-batch-beta-testers'
+    batch_beta_tester_selector = '.batch-beta-testers'
+
+    def is_browser_on_page(self):
+        return self.q(css=self.batch_beta_tester_heading_selector).present
+
+    def fill_beta_tester_text_box(self, username):
+        """
+        Fill in the form with the provided username and submit it.
+        """
+        username_selector = "{} textarea".format(self.batch_beta_tester_selector)
+        enrollment_button = "{} .enrollment-button[data-action='add']".format(self.batch_beta_tester_selector)
+
+        # Fill the username  after the username selector is visible.select_auto_enroll_section
+        self.wait_for_element_visibility(username_selector, 'username field is visible')
+        self.q(css=username_selector).fill(username)
+
+        # Verify enrollment button is present before clicking
+        EmptyPromise(
```

Comment:
Yeah there is an other way, but here i am just checking if the element is preset before clicking and i am checking it by the `EmptyPromise `.